blendergltf.py

```diff
@@ -250,7 +250,12 @@ def export_buffer(self, settings):
         if settings['buffers_embed_data']:
             uri = 'data:text/plain;base64,' + base64.b64encode(data).decode('ascii')
         else:
-            uri = bpy.path.clean_name(self.name) + '.bin'
+            if settings['buffers_combine_data']:
+                parts = os.path.splitext(os.path.basename(settings['filepath']))
```

Comment:
In my original "separate buffers" PR, the geo data was preserved as a bytearray all the way through export_gltf. It was passed through to __init__, so a module could make use of it, and you don't need to pass in the filename. Would this be a better solution? Makes for a more complicated PR, but cleaner code I think.